src/main.rs

```diff
@@ -188,7 +188,7 @@ fn install_hook(color_scheme: ColorScheme) -> Result<(), Box<dyn Error>> {
         Err(e) if e.kind() == std::io::ErrorKind::AlreadyExists => {
             eprintln!(
                 "Failed to create `{HOOK_PATH}` as it already exists. \
-                Please either or remove it and re-run `gimoji -i` or \
+                Please either remove it and re-run `gimoji -i`, or \
                 add the following commandline to it:\n{content}",
```

Comment:
"command line"

Reply:
or "command-line" according to [wikipedia](https://en.wikipedia.org/wiki/Command-line_interface).